Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp

```diff
@@ -299,13 +299,6 @@ void WebsiteDataStore::resolveDirectoriesIfNecessary()
     if (!m_configuration->modelElementCacheDirectory().isEmpty())
         m_resolvedConfiguration->setModelElementCacheDirectory(resolveAndCreateReadWriteDirectoryForSandboxExtension(m_configuration->modelElementCacheDirectory()));
 #endif
-#if PLATFORM(IOS_FAMILY)
```

Comment:
Can you explain this change? It is not obvious to me and not explained in the change log I think.

Reply:
This code stores WebProcessPool::cookieStorageDirectory() to WebsiteDataStore ::m_cookieStorageDirectory. As we moved WebProcessPool::cookieStorageDirectory() to WebsiteDataStore::cookieStorageDirectory(), WebsiteDataStore doesn't need the variable. And we will resolve directory when creating sandbox extension handle (createHandleForReadWriteDirectory).